WordPress/WordPressUITests/Screens/Editor/EditorPostSettings.swift

```diff
@@ -6,6 +6,9 @@ class EditorPostSettings: BaseScreen {
     let categoriesSection: XCUIElement
     let tagsSection: XCUIElement
     let featuredImageButton: XCUIElement
+    var changeFeaturedImageButton: XCUIElement {
+        return settingsTable.cells["CurrentFeaturedImage"]
+    }
```

Comment:
Nitpick: Any reason why this is declared this as a `var` like this instead of `let` like the other elements on the screen? Not a huge deal but it would be nice to be consistent.

Reply:
It's a var because it's a dynamic variable, every time you access it, it returns a new value.
I need to do this because the cell is not available when you go in the screen, only after you set an featured image.